crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -81,4 +82,10 @@ struct FragmentInput {
 fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
     return vec4(in.world_normal * 0.5 + vec3(0.5), 1.0);
 }
+
+#else // NORMAL_PREPASS
+
+@fragment
+fn fragment(in: VertexOutput) {}
```

Comment:
I'm not sure I understand why this is necessary. The pipeline sets the fragment state to None if the normal prepass is not enabled.

Reply:
This was previously the case, yes, although not exactly. Before my changes, the fragment state was set if either the normal prepass was enabled or the material used an alpha mask. I believe, if the `CustomMaterial` in the `shader_prepass` example would have used `AlphaMode::Mask` as `alpha_mode`, it would break even without my changes. 
Now the fragment state will also be set if the alpha mode is `Blend`, `Add` or `Premultiplied`. This triggers the issue with the missing entry point because the `CustomMaterial` uses `AlphaMode::Blend`.

Reply:
I can confirm my suspicion, going back to a39c22386b0eadd5c39e342a2ce9dd4d920bc492 right before the merge of #7784 and changing the alpha mode to `AlphaMode::Mask(0.5)` causes the example to fail with:
```
thread 'main' panicked at 'wgpu error: Validation Error

Caused by:
    In Device::create_render_pipeline
      note: label = `prepass_pipeline`
    error matching FRAGMENT shader requirements against the pipeline
    location[0] is provided by the previous stage output but is not consumed as input by this stage.

'
```
Interestingly, it doesn't complain about a missing fragment entry point, which is what I expected.